### ISSUE TITLE
The LGC in you code can not drop the number of parameters during processing. 

### ISSUE THREAD

**Issue by liminle** [markdtw/condensenet-tensorflow#4]
Hi markdtw,
Thank you for the share!
I have tested your code and found that there was no drop in the number of parameters.
I set the epoch=4, and the parameter numbers are 1.43M after all of the four epochs. 
So, it seems that the LGC in the code worded just like standard CONV.
It confused me.

**Comment by markdtw** [comment]
Hi!

The "pruning" process in this implementation is just masking out parameters with another tensor. The actual re-grouping requires an additional step to shuffle the weights based on the mask tensor (In the author's Pytorch implementation, this is done at the post-processing step as well). So the number of parameters is still the same unless you specifically exclude those "masked-out" parameters.
As stated in the README, this phase is not implemented yet. Any help will be appreciated :)

**Comment by liminle** [comment]
Thank you!
